reference/commands/consumer/config.rst

```diff
@@ -200,3 +203,20 @@ Conan runs it based on *config_install.json*, including the timestamp of the las
   .. code-block:: bash
 
       $ conan config install /path/to/some/config.zip
+
+- List all previously installed origins (the ones that will executed if :command:`conan config install` is called without args):
+
+  .. code-block:: bash
+
+      $ conan config install --list
+
+This will display the list of stored origins, with their index inside the list.
+
+- Remove one of the previously installed origins:
+
+  .. code-block:: bash
+
+      $ conan config install --remove=1
+
+This will remove the element with index=1 (second element in the list) of the existing origins. Future :command:`conan config install`
+or scheduled updates will not be using this origin anymore.
```

Comment:
```suggestion
This will remove the element with index=1 (second element in the list) of the existing origins. This means that the next
:command:`conan config install` manual or scheduled calls to this command will not use this origin anymore.
```